docs/editor/accessibility.md

```diff
@@ -70,179 +70,181 @@ In `settings.json` file, nest the following code inside the outermost curly brac
 
 ### Selecting accessible colors
 
-The accessibility of colors is subjective to the type of anomalous trichromacy (color blindness). The level of severity ranges per person and can be divided into four condition types:
+The accessibility of colors is subjective to the type of anomalous trichromacy (colorblindness). The level of severity ranges per person and can be divided into four condition types:
 
 |   Condition      |   Type    |
 |       ---        |    ---    |
-|   Deuteranopia   |   Defined by the reduced sensitivity to green light. It is the most common form of color blindness.
-|   Protanopia     |   Defined by the reduced sensitivity to red light.
-|   Tritanopia     |   Defined by the reduced sensitivity to blue light. This condition is considered rare.
-|   Monochromia    |   Also referred to as, achromatopsia and is defined by the inability to see all colors. This is the rarest form of color blindness. Go to [Foundation for Fighting Blindness](https://www.fightingblindness.org/diseases/achromatopsia) for more information.
+|   Deuteranopia   |   Reduced sensitivity to green light. It is the most common form of color blindness.
+|   Protanopia     |   Reduced sensitivity to red light.
+|   Tritanopia     |   Reduced sensitivity to blue light. This condition is considered rare.
+|   Monochromia    |   The inability to see all colors, also referred to as *achromatopsia*. More information about the rarest form of colorblindness: [Foundation Fighting Blindness](https://www.fightingblindness.org/diseases/achromatopsia).
 
 One of the best approaches to selecting the best colors for a specific condition is to apply complementary colors. These are colors located opposite of one another on a color wheel.
 
-![A color wheel highlighting complementary colors for regular vision, deuteranopia, protanopia, tritanopia and monochromacy](images/accessibility/accessibility-color-wheels.png)
+![Color wheel highlighting complementary colors for regular vision, deuteranopia, protanopia, tritanopia and monochromacy](images/accessibility/accessibility-color-wheels.png)
 
->**Note**: For more information on finding complementary colors, go to [Adobe Color](https://color.adobe.com/create/color-accessibility) to access the color blind simulator and interactive color wheel.
+>**Note**: For more information on finding complementary colors, access the colorblind simulator and interactive color wheel at [Adobe Color](https://color.adobe.com/create/color-accessibility).
 
 ## Dim unfocused editors and terminals
 
-Unfocused views can be dimmed to make it clearer where typed input will go. This is especially useful when working with multiple editor groups or terminals. Enable this feature by setting `"accessibility.dimUnfocused.enabled": true`. You can control the dimness level with `accessibility.dimUnfocused.opacity`, which takes the opacity fraction from 0.2 to 1 (default 0.75).
```

Comment:
Why remove the section on the opacity fraction values?

Reply:
Good catch—that might have been a mistake. I partially rewrote the instruction portion and meant to circle back with you about how to handle the opacity fraction instructions, which didn't fully explain how to do the thing. In my local copy, that paragraph appears this way: 

Unfocused views can be dimmed to make it more clear where typed input will go. This is especially useful when working with multiple editor groups or terminals. Turn this feature on by setting `"accessibility.dimUnfocused.enabled": true` and configure the dimness level with `accessibility.dimUnfocused.opacity`.

Reply:
I don't like putting these two settings in one sentence as to me it implies that the reader needs to do both. 
We like users to enable the main setting and don't expect many to adjust the opacity from the default.
I'm also not sure about switching "clearer" to "more clear". Why do you think the change is necessary?